pkg/kopia/command/parse_command_output_test.go

```diff
@@ -121,7 +121,32 @@ func (kParse *KopiaParseUtilsTestSuite) TestSnapshotInfoFromSnapshotCreateOutput
 		{
 			output: `Snapshotting u2@h2:/tmp/aaa1 ...
 			* 0 hashing, 1 hashed (2 B), 3 cached (4 B), uploaded 5 KB, estimating...
-		   {"id":"00000000000000000000001","source":{"host":"h2","userName":"u2","path":"/tmp/aaa1"},"description":"","startTime":"2021-05-26T05:29:07.206854927Z","endTime":"2021-05-26T05:29:07.207328392Z","rootEntry":{"name":"aaa1","type":"d","mode":"0755","mtime":"2021-05-19T15:45:34.448853232Z","obj":"ka68ba7abe0818b24a2b0647aeeb02f29","summ":{"size":0,"files":1,"symlinks":0,"dirs":1,"maxTime":"2021-05-19T15:45:34.448853232Z","numFailed":0}}}
+			{
```

Comment:
I may be wrong, but it seems that this test will fail.

Reply:
@julio-lopez let me resolve merge confllict

Reply:
@julio-lopez @julio-lopez It looks like the changes that i made is to log file and which will be changed every time . So, enabling this linter does not make sense.  what do you think ?

Reply:
Have you run the test?

Reply:
sorry , Entire test cases or `golint` only . linter test case has passed .

Reply:
Tests as well, it looks like those are going to fail

Reply:
yeah, `make test`  went to failure .

```
ok  	github.com/kanisterio/kanister/pkg/virtualfs	0.011s
FAIL
real 360.99
user 0.05
sys 1.52
make[1]: *** [Makefile:217: run] Error 1
make[1]: Leaving directory '/home/anishbista/kanister'
make: *** [Makefile:165: go-test] Error 2
```

But  honestly unable to figure out the reason behind it . 
could you run the test cases here ?

Reply:
Hi @anishbista60,
> could you run the test cases here ?

If it's failing for you, it's going to fail for anyone else who tries to run the test.

> But honestly unable to figure out the reason behind it .

do you need help figuring out why the test is failing? Do you understand what exactly the test is doing?